src/app/page.tsx

```diff
@@ -1,9 +1,137 @@
+          >
+            Volunteer Now
+          </Button>
+        </ButtonDiv>
+      </TitleSection>
+      <StatisticsSection>
+        <StatContainer>
+          <StatNumber>100+</StatNumber>
+          <StatLabel>Matched Cases</StatLabel>
+        </StatContainer>
+        <StatContainer>
+          <StatNumber>50+</StatNumber>
+          <StatLabel>Attorneys</StatLabel>
+        </StatContainer>
+        <StatContainer>
+          <StatNumber>100+</StatNumber>
+          <StatLabel>Translation Support</StatLabel>
+        </StatContainer>
+        <StatContainer>
+          <StatNumber>100+</StatNumber>
+          <StatLabel>Succesful Outcomes</StatLabel>
+        </StatContainer>
+      </StatisticsSection>
+      <HowYouCanHelpContainer>
+        <Title>HOW YOU CAN HELP</Title>
+        <Subtitle>
+          The Immigration Justice Project seeks to promote due process and
+          access to justice for indigent immigrants and asylum seekers.
+        </Subtitle>
+        <ServicesDiv>
+          <ServiceContainer>
+            <ServiceIcon
+              src="path_to_case_assistance_icon"
+              alt="An Icon Will Be here"
+            />
+            <ServiceTitle>Case Assistance</ServiceTitle>
+            <ServiceDescription>
+              Attorneys licensed in any state with or without immigration law
+              experience can represent immigrants and asylum-seekers in removal
+              proceeding before the Immigration Court, the Board of Immigration
+              Appeals and the U.S. Court of Appeals for the Ninth Circuit.
+            </ServiceDescription>
+          </ServiceContainer>
+          <ServiceContainer>
+            <ServiceIcon
+              src="path_to_case_assistance_icon"
+              alt="An Icon Will Be here"
+            />
+            <ServiceTitle>Limited Case Assistance</ServiceTitle>
+            <ServiceDescription>
+              Law students and recent graduates awaiting bar results can assist
+              attorneys to screen potential clients, conduct legal and factual
+              research and write motions and briefs in support of on-going court
+              cases.
+            </ServiceDescription>
+          </ServiceContainer>
+          <ServiceContainer>
+            <ServiceIcon
+              src="path_to_case_assistance_icon"
+              alt="An Icon Will Be here"
+            />
+            <ServiceTitle>Translation Assignment</ServiceTitle>
+            <ServiceDescription>
+              Interpreters and translators can conduct volunteer translation and
+              live interpretation. We have frequent need for individuals who
+              speak Creole, Portuguese, Spanish, French, Arabic, and Russian.
+            </ServiceDescription>
+          </ServiceContainer>
+        </ServicesDiv>
+      </HowYouCanHelpContainer>
+      <MissionValuesContainer>
+        <img src={image.src} alt="Smaller Stock" width={521} height={338} />
```

Comment:
I would recommend using the [Image component](https://nextjs.org/docs/pages/api-reference/components/image) provided by Next.js to help optimize it

Reply:
Ideally, we shouldn't fix arbitrary pixel values to width and height, but rather use units that help scale it relative to the viewport. In this case, I would recommend using a `grid` layout and specifying a reasonable fractional unit for the left column.